src/TypeInference/Error.ml

```diff
@@ -286,13 +289,32 @@ let ctor_not_in_type ~pos ~env name tp =
     (Pretty.type_to_string pp_ctx env tp)
   in (pos, msg ^ Pretty.additional_info pp_ctx, [])
 
+let escaping_tvar_message ~env x =
+  let pp_ctx = Pretty.empty_context () in
+    let msg = Printf.sprintf
```

Comment:
Code formatting could be improved. Let-definitions should be on the same indentation level.